File: Mongo/mongodb_nbp/Backend/MongoDB/MongoDB/Models/Narudzbina.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;

namespace MongoDB.Models
{
    public class Narudzbina
    {
        public ObjectId id { get; set; }
        public string id_string { get; set; }
        public Korisnik klijent { get; set; }
        public List<Proizvod> proizvodi { get; set; }
        public float cena { get; set; }
        [BsonRepresentation(BsonType.String)]
        public Status status { get; set; }
        public DateTime vreme { get; set; }
    }
}

File: Mongo/mongodb_nbp/Frontend/Tabemono/src/models/Narudzbina.ts
import { Time } from '@angular/common';
import ObjectID from 'bson-objectid';
import { Status } from 'src/enums/enums';
import { korisnik } from './Korisnik';
import { proizvod } from './Proizvod';

export interface narudzbina {
  id: ObjectID;
  id_string: string;
  klijent: korisnik;
  proizvodi: proizvod[];
  cena: number;
  status: Status;
  vreme: Time;
}
